src/app/cli/src/mina.ml

```diff
@@ -326,9 +326,9 @@ let setup_daemon logger =
         "true|false Generate a new genesis proof for the current \
          configuration if none is found (default: false)"
       (optional bool)
-  and disable_telemetry =
-    flag "--disable-telemetry" ~aliases:["disable-telemetry"] no_arg
-      ~doc:"Disable reporting telemetry to other nodes"
+  and disable_node_status =
+    flag "--disable-node-status" ~aliases:["disable-node-status"] no_arg
+      ~doc:"Disable reporting node status to other nodes"
```

Comment:
should we mention that it is enabled by default? I don't think we mention it anywhere

Reply:
added `(default: enabled)` to docstring.